### ISSUE TITLE
1.5.1.4  DH  CDH count error

### ISSUE THREAD

**Issue by wingchaos** [ravahn/FFXIV_ACT_Plugin#91]
1.5.1.4 Healer 's DirectHitCount &DirectHitPct&CritDirectHitCount&CritDirectHitPct do not work normally...All them display "0" .

**Comment by Squall-Leonhart** [comment]
DirectHit types are only working for healer in 50 content.

**Comment by christianallred** [comment]
Check for opened issues before submitting. I literally posted this 6 hours before yours

**Comment by christianallred** [comment]
@Squall-Leonhart  are the calculations correct? or are oyu simply saying you aren't seen the numbers at all when not Level synced to 50?

**Comment by ravahn** [comment]
Heals do not Direct Hit in FFXIV at this point in time, so those columns only report damage data.

There are some problems with the columns, but these are documented in issue #90 so closing this.

**Comment by Squall-Leonhart** [comment]
healer attacks do directhit.....
and do not register in ACT.

**Comment by ravahn** [comment]
I just performed 600 autoattacks as a level 15 cnj.  My crit rate was 6%, which is close to the correct value of 5% (confirmed by using a DoT and looking at the DoT crit rate from the effect info).  My DH rate was 0.0%, confirmed by searching the chatlog log for all of my autoattacks.

Either the game has a bug and doesn't report DH hits it to the client, or healers before a certain level cannot DH even for damage attacks.

I performed a limited test with lv 15 Mrd - 100 swings - and confirmed the same thing - zero DHs, but correct critical hit rate.  This suggests this is not a healer-specific behavior, but is level-dependent.  Perhaps DH is limited to players above level 50, or some other level cutoff.

Either way, this is not a FFXIV Plugin issue, it is the behavior of the game.
